magit-annex.el

```diff
@@ -499,10 +503,11 @@ arguments don't include --from."
                       (list file))
                     (and (derived-mode-p 'dired-mode)
                          (dired-get-marked-files t))))
-         (default (mapconcat #'identity files ",")))
+         (default (mapconcat #'identity files "|"))
+         (crm-separator "[ 	]*|[ 	]*"))
```

Comment:
I know this issue exists in the default crm-separator, but since we're defining a custom one here, I think it'd be good to drop the spaces from the regexp to support names with stupidly positioned spaces.  At first I thought that might come at the expense of not helping users that accidentally added leading or trailing spaces around the separator, but completing-read-multiple doesn't make it easy for a user to do that.